crml/cennzx-spot/src/impls.rs

```diff
@@ -40,17 +40,13 @@ where
 	fn exchange_address_for(core_asset_id: T::AssetId, asset_id: T::AssetId) -> T::AccountId {
 		let mut buf = Vec::new();
 		buf.extend_from_slice(b"cennz-x-spot:");
-		buf.extend_from_slice(&u64_to_bytes(core_asset_id.into()));
-		buf.extend_from_slice(&u64_to_bytes(asset_id.into()));
+		buf.extend_from_slice(&core_asset_id.into().to_le_bytes());
```

Comment:
I don't think this is a good idea - AssetID is a u32.
Originally, this would exend buf by 8 bytes, this change extends it by 4.

Reply:
`.into()` makes it into a `u64` so the behaviour should be the same, just one less indirection